spec/spec_helper.rb

```diff
@@ -61,11 +61,14 @@
 OmniAuth.config.test_mode = true
 
 Capybara.register_driver(:cuprite) do |app|
+  timeout = is_ci ? 10 : 999_999
```

Comment:
This allows for a debugging session with headful Chrome that won't be interrupted very quickly by Chrome being closed.